pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -257,40 +238,10 @@ qtModule ({
 
   # FIXME These dependencies shouldn't be needed but can't find a way
   # around it. Chromium pulls this in while bootstrapping GN.
-  ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    libobjc
-    cctools
-
-    # frameworks
-    ApplicationServices
-    AVFoundation
-    Foundation
-    ForceFeedback
-    GameController
-    AppKit
-    ImageCaptureCore
-    CoreBluetooth
-    IOBluetooth
-    CoreWLAN
-    Quartz
-    Cocoa
-    LocalAuthentication
-    MediaPlayer
-    MediaAccessibility
-    SecurityInterface
-    Vision
-    CoreML
-    OpenDirectory
-    Accelerate
-
-    openbsm
-    libunwind
-  ];
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [ cctools.libtool ];
```

Comment:
This comment seems probably inaccurate now? Though I don’t really know what it meant to begin with.

Reply:
I’m seeing if I can drop it now that xcbuild includes `xcrun`. I also tried dropping the patches as suggested above.

Reply:
Ah, I meant this comment that I don’t really understand:
```nix
  # FIXME These dependencies shouldn't be needed but can't find a way
  # around it. Chromium pulls this in while bootstrapping GN.
```

Reply:
I’m seeing if I can go one step further and delete the whole thing. If not, I’ll drop or change the comment.

Reply:
Nope. Deleting those would be non-trivial. It requires replacing the qtbase mkspecs patches with different patches to use xcbuild. If someone wants to do that later, that’s fine; but it’s beyond the scope of what I want to do with this.